prometheus-exporters/vrops-exporter/alerts/vcenter.alerts

```diff
@@ -2,8 +2,9 @@ groups:
 - name: vcenter.alerts
   rules:
   - alert: VcsaSslCertificateExpiryWarning
-    expr: >
-      vrops_vcenter_vcsa_certificate_remaining_days < 31 and vrops_vcenter_vcsa_certificate_remaining_days > 15
+    expr: |
+      (vrops_vcenter_summary_certificate_expiry_date / 1000) - time() < (31*86400) 
+      and on(vcenter) (vrops_vcenter_summary_certificate_expiry_date / 1000) - time() > (15*86400)
```

Comment:
Sophisticated. But I would rather stay with days like so:
```suggestion
    expr: 15 > round(((vrops_vcenter_summary_certificate_expiry_date / 1000) - time()) / 86400) < 31
```

Reply:
I like it!